src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -108,6 +111,37 @@ export class Main extends Component {
     this.props.toggleMobileDisplayHidden(hidden);
   };
 
+  focusTrap = e => {
+    const buttonContainer = document.getElementById('va-nav-controls');
+    const megaMenuContainer = document.getElementById('mega-menu-mobile');
+    const focusable = [
+      ...Array.from(buttonContainer.querySelectorAll(tabbableSelectors)).filter(
+        el =>
+          el.getAttribute('tabindex') !== '-1' &&
+          el.getAttribute('hidden') === null,
+      ),
+      ...Array.from(
+        megaMenuContainer.querySelectorAll(tabbableSelectors),
+      ).filter(
+        el =>
+          el.getAttribute('tabindex') !== '-1' &&
+          el.getAttribute('hidden') === null,
+      ),
+    ];
+    const firstEl = focusable[0];
+    const lastEl = focusable[focusable.length - 1];
+
+    if (e.keyCode === 9) {
```

Comment:
Up to you of course but do you think it reads better if variables are fully spelled out? e.g. `event`, `firstElement`, etc.?